kvbc/include/ClientImp.h

```diff
@@ -35,6 +35,8 @@ class ClientImp : public IClient {
                                     uint32_t* outActualReplySize,
                                     const std::string& = "") override;
 
+  virtual void setMetricsAggregator(std::shared_ptr<concordMetrics::Aggregator> aggregator) override;
```

Comment:
minor: don't use 'virtual' when overriding

Reply:
All other methods in this class are defined that way.